gui.py

```diff
@@ -1,6 +1,7 @@
 #!/usr/bin/env python3
 
 import tkinter as tk
+import tkinter.simpledialog as dialog
```

Comment:
This should be replaced by a custom module made with just tk. This should be a semi-high-priority change.